canonical/juju-dashboard
# WD-5856 - Merge cross model queries to main

## Done

- Merge the cross-model-queries feature branch into main.

## QA

- Check that the cross model queries features work when connected to staging.
- Check that the cross model queries page does not appear when using a local controller or JAAS.

## Details

https://warthogs.atlassian.net/browse/WD-5856
